frontend/epfl-google-forms/controller.php

```diff
@@ -27,7 +27,7 @@ function epfl_google_forms_block( $attributes ) {
     $url_host = parse_url($url, PHP_URL_HOST);
 
     /* Check that iframe has a Google host as source */
-    if(preg_match('/\.google\.com$/', $url_host) !== 1)
+    if($url_host == null || preg_match('/\.google\.com$/', $url_host) !== 1)
```

Comment:
Pour des questions de lisibilité, je préfère ce format. 
A ta discrétion.
```suggestion
    if (
        $url_host == null ||
        preg_match('/\.google\.com$/', $url_host) !== 1
    )
```